packages/mdc-slider/_mixins.scss

```diff
@@ -20,116 +20,457 @@
 // THE SOFTWARE.
 //
 
+@import "@material/feature-targeting/functions";
+@import "@material/feature-targeting/mixins";
 @import "@material/theme/functions";
 @import "@material/theme/mixins";
 @import "@material/theme/variables"; // for mdc-theme-prop-value
+@import "@material/typography/mixins";
+@import "@material/rtl/mixins";
 @import "./variables";
+@import "./keyframes";
 
 //
 // Public
 //
+@mixin mdc-slider-core-styles($query: mdc-feature-all()) {
+  $feat-structure: mdc-feature-create-target($query, structure);
+  $feat-animation: mdc-feature-create-target($query, animation);
 
-@mixin mdc-slider-highlight-color($color) {
+  // postcss-bem-linter: define slider
+
+  @include mdc-feature-targets($feat-animation) {
+    @include mdc-slider-emphasize-keyframes_;
+  }
+
+  .mdc-slider {
+    @include mdc-slider-color-accessible(secondary, $query);
+
+    &--disabled {
+      $disabled-color: #9a9a9a;
+
+      @include mdc-slider-highlight-color_($disabled-color, $query);
+      @include mdc-slider-rail-color_($disabled-color, $query: $query);
+      @include mdc-slider-rail-tick-mark-color_($disabled-color, $query);
+      @include mdc-slider-thumb-color_($disabled-color, $query);
+      @include mdc-slider-thumb-stroke-cutout_(white, $query);
+
+      @include mdc-feature-targets($feat-structure) {
+        cursor: auto;
+      }
+    }
+
+    @include mdc-feature-targets($feat-structure) {
+      position: relative;
+      width: 100%;
+      height: 48px;
+      cursor: pointer;
+      touch-action: pan-x;
+      -webkit-tap-highlight-color: rgba(black, 0);
+
+      &:focus {
+        outline: none;
+      }
+    }
+
+    @include mdc-slider-track_($query);
+    @include mdc-slider-track-marker_($query);
+    @include mdc-slider-thumb_($query);
+    @include mdc-slider-focus-ring_($query);
+    @include mdc-slider-pin_($query);
```

Comment:
Note that these were all under one huge selector. I broke them up into mixins so it's easier to follow.

Reply:
Thanks!